src/Compilers/Core/Portable/Symbols/ITypeSymbol.cs

```diff
@@ -111,6 +111,26 @@ public interface ITypeSymbol : INamespaceOrTypeSymbol
         /// has no concept of unmanaged types.
         /// </summary>
         bool IsUnmanagedType { get; }
+
+        // PROTOTYPE(nullable-api): Document
+        string ToDisplayString(Nullability topLevelNullability, SymbolDisplayFormat format = null);
```

Comment:
>string ToDisplayString(Nullability topLevelNullability, SymbolDisplayFormat format = null); [](start = 8, length = 91)

Does this method need to be included in the interface or could it be defined as an extension method that returns:
```
type.ToDisplayParts(nullability, format).ToDisplayString();
```

Reply:
Could it be? Yes, but that would be inconsistent with way that the existing `ToDisplay(Minimal)DisplayString` works today.

---
In reply to: [259153684](https://github.com/dotnet/roslyn/pull/33531#discussion_r259153684) [](ancestors = 259153684)